js/apps/system/_admin/aardvark/APP/frontend/js/views/databaseView.js

```diff
@@ -176,34 +176,39 @@
       if (frontendConfig.forceOneShard) {
         sharding = 'single';
       }
-      var replicationFactor = $('#new-replication-factor').val();
-      var writeConcern = $('#new-write-concern').val();
-
-      var options = {
-        name: dbname,
-        "options" : {
-          "sharding" : sharding,
-          "replicationFactor" : Number(replicationFactor),
-          "writeConcern" : Number(writeConcern),
-        },
-        users: [{
-          username: userName
-        }]
-      };
-
-      this.collection.create(options, {
-        error: function (data, err) {
-          self.handleError(err, dbname);
-        },
-        success: function (data) {
-          if (window.location.hash === '#databases') {
-            self.updateDatabases();
+      
+      let replicationFactor = $('#new-replication-factor').val();
+      let writeConcern = $('#new-write-concern').val();
+
+      if(parseInt(writeConcern) <= parseInt(replicationFactor)) {
```

Comment:
What happens in case replicationFactor || writeConcern will not be typeof String? 
(Before it was the same - but I think we should validate input here) - Try / catch would help I guess.